web/forms.py

```
@@ -1802,3 +1804,21 @@ class StudyGroupForm(forms.ModelForm):
+
+
+class SurveyForm(forms.ModelForm):
+    title = forms.CharField(
+        max_length=200,
+        widget=TailwindInput(attrs={"placeholder": "Enter survey title"}),
+        help_text="Give your survey a clear and descriptive title",
+    )
+
+    class Meta:
+        model = Survey
+        fields = ["title"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)